### PR TITLE
Fixed auto save being triggered after loading a save or first entering

### PR DESCRIPTION
**Pull request by hhyyrylainen**
the microbe stage

by making an explicit property checks for when the stage fade in is complete, and the stage has returned from the editor.

### CODE REVIEW

**Review by Satharis — APPROVED** [review]
Works properly from my testing.

**Comment by hhyyrylainen** [comment]
I noticed some problems that I fixed.

File: src/microbe_stage/MicrobeStage.cs [hunk 1]
```diff
@@ -134,6 +139,12 @@ public List<Node> SavedGameEntities
         set => savedGameEntities = value;
     }
 
+    /// <summary>
+    ///   True once stage fade-in is complete
+    /// </summary>
+    [JsonIgnore]
+    public bool TransitionFinished { get; internal set; }
```
Comment (Kasterisk) [inline src/microbe_stage/MicrobeStage.cs:146]: Is this functionally the same as { get; set; }?

**Review by Kasterisk — APPROVED** [review]
Yeah it seems good

File: src/microbe_stage/MicrobeStage.cs (hunk 1 above)
Comment (hhyyrylainen) [reply src/microbe_stage/MicrobeStage.cs:146]: Basically yeah. If we had multiple modules, only the code in the same module can call internal stuff. So I used that here to mark that not everyone should use the setter.

**Pull request merged**